### ISSUE TITLE
Terraform binary is required to debug the extension

### ISSUE THREAD

**Issue by glennsarti** [hashicorp/vscode-terraform#622]
While trying to develop the extension I followed the instructions in `DEVELOPMENT.MD`.  Pressing F5 to start a debug session just hung. Turns out there were two issues.

1. In `.vscode/tasks.json` the default `"reveal"` is all set to `never`.  So I could never see why it failed

2. The debug "Launch Client" configuration calls the "npm watch" task.  However, there are actually two tasks defined for "npm watch" in `tasks.json`.  The one you'd expect, and `initAndWatch`.  I'm guessing the debug configuration took the last defined one (`initAndWatch`) and tried to run that, which tried to call `terraform init` and failed because I did not have terraform installed.

Bottom line, the development instructions should be updated to include that terraform is a required piece of software for development OR the `tasks.json` file should be cleaned up so the correct task gets called instead.

This is not an issue with the extension per-se, it's an onboarding developers issue.
